charts/osm/templates/osm-deployment.yaml

```diff
@@ -70,6 +70,11 @@ spec:
               scheme: HTTP
               path: /health/alive
               port: 9091
+          env:
```

Comment:
IMO I'd avoid introducing an `env` instead of an `arg` if this is a mandatory flag; now the user has to care about both `args` and `envs`.

Is that possible though...? Not sure the value of `metadata.name` will be available to set as arg

Reply:
What user?

The pod name is not known at deployment time, and using env is a straightforward way to dynamically populate this when the container is set up.

Reply:
On second thought, this can't be a configuration either after all. I was contemplating the case of someone having issues with this variable not being set for whatever reason. Can't be set by the user anyway.